src/Util/Crypto.php

```diff
@@ -313,6 +313,6 @@ public static function randomDigits($digits)
 			$rn .= random_int(0, 9);
 		}
 
-		return $rn;
+		return (int) $rn;
```

Comment:
Since you're editing this function, its whole body should be replaced with 
```php
return random_int(0, 10 ** $digits - 1);
```
This would probably make the tests for it moot since we're directly relying on a native PHP function.

Reply:
I made the change and refactored the tests.